### PR TITLE
Add loading and no internet connection error views

### PR DESCRIPTION
**Pull request by matir91**
## Changes proposed in this request:
* Add a default theme
* Add a toolbar with a close button 
* Add a no internet error view
* Add spinner 

## Preview
anim

### CODE REVIEW

File: me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/WebViewActivity.java [hunk 1]
```diff
@@ -18,23 +29,115 @@
...
+          });
+    }
+  }
+
+  private void hideInternetConnectionErrorView(int startDelayInSeconds) {
+    if (progressBarWebView != null) {
+      internetConnectionErrorView.animate()
+          .alpha(0.0f)
+          .setDuration(500)
+          .setStartDelay(startDelayInSeconds * 1000)
```
Comment (m-revetria) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/WebViewActivity.java]: is `1000` ok here? in the previous function you are using the same value passed to `duration`

File: me.id.webverify/webverifylib/src/main/res/layout/activity_web_view.xml [hunk 2]
```diff
@@ -1,18 +1,63 @@
...
+
+    <WebView
+        android:id="@+id/progress_bar_webView"
+        android:layout_width="56dp"
+        android:layout_height="56dp"
+        android:layout_centerInParent="true"
         />
 
-</RelativeLayout>
+</RelativeLayout>
```
Comment (m-revetria) [inline me.id.webverify/webverifylib/src/main/res/layout/activity_web_view.xml]: missing empty line at EOF

File: me.id.webverify/webverifylib/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="blue">#ff2e3d50</color>
-    <color name="green">#2fc073</color>
+    <color name="idme_black">#2d3E51</color>
+    <color name="idme_blue">#3aabe0</color>
+    <color name="idme_blue_dark">#0d94B9</color>
+    <color name="idme_white_dark">#f1f1f1</color>
 </resources>
```
Comment (m-revetria) [inline me.id.webverify/webverifylib/src/main/res/values/colors.xml]: missing empty line at EOF

File: me.id.webverify/webverifylib/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
-    <string name="app_name">Webverifylib</string>
+    <string name="id_me_sdk_app_name">ID.me Wallet</string>
 
     <string name="idme_web_verify_get_common_uri"><![CDATA[https://api.id.me/oauth/authorize]]></string>
     <string name="idme_web_verify_get_access_token_uri"><![CDATA[https://api.id.me/oauth/token]]></string>
     <string name="idme_web_verify_get_profile_uri">https://api.id.me/api/public/v2/data.json</string>
+    <string name="close">Close</string>
+    <string name="no_internet_connection">No internet connection</string>
+    <string name="unavailable">Unavailable</string>
+    <string name="id_me_sdk_requires_internet_connection">ID.me Wallet requires a \n connection to the internet.</string>
```
Comment (m-revetria) [inline me.id.webverify/webverifylib/src/main/res/values/strings.xml]: this was changed to _ID.me Wallet requires an internet connection_

File: me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/WebViewActivity.java (hunk 1 above)
Comment (matir91) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/WebViewActivity.java]: yes, I need convert seconds to milliseconds.

File: me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/WebViewActivity.java [hunk 5]
```diff
@@ -167,7 +164,8 @@ public void onDestroy() {
   protected void clearWebViewCacheAndHistory() {
     webView.clearCache(true);
     webView.clearHistory();
-    webView.clearFormData();
+    webView.clearFormData();;
```
Comment (sdownie) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/WebViewActivity.java]: @matir91  Should this have a double ;;?

Comment (matir91) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/WebViewActivity.java]: no! my bad!

**Comment by matir91** [comment]
@sdownie can you review it again ?

**Pull request merged**